src/pages/docs/octopus-ai/mcp/use-cases.mdx

```diff
@@ -7,13 +7,18 @@ description:
 navOrder: 1
 ---
 
+Below are some typical example use cases that might give you a good starting point to experiment with the Octopus Deploy MCP server.
+
+We are eager to hear how you use it and what features you would like to see included in future versions.
+
 ## Capability: Change Management
 
 Reason about what changes have been deployed where, to help you understand what your customers are using in production.
 
-Example: Quickly find out what version of your software a customer, represented by a Tenant, is running in Production, and identify if there were any issues with their most recent deployment.
+### Production Version Tracking
+Quickly find out what version of your software a customer, represented by a Tenant, is running in Production, and identify if there were any issues with their most recent deployment.
 
-### 📝 Prompt
+#### 📝 Example Prompt
```

Comment:
Your document outline will have many of the same headings, is it possible to make each unique. For example: "Example version tracking prompt"?